src/requests/cache/sorted_set_put_elements.rs

```diff
@@ -0,0 +1,105 @@
+use momento_protos::cache_client::{SortedSetElement, SortedSetPutRequest};
+
+use crate::requests::cache::MomentoRequest;
+use crate::simple_cache_client::prep_request_with_timeout;
+use crate::{CacheClient, CollectionTtl, IntoBytes, MomentoResult};
+
+/// Request to add elements to a sorted set. If an element already exists, its score is updated.
+/// Creates the sorted set if it does not exist.
+///
+/// # Arguments
+///
+/// * `cache_name` - The name of the cache containing the sorted set.
+/// * `sorted_set_name` - The name of the sorted set ot add an element to.
+/// * `elements` - The values and scores to add. The values must be able to be converted to a Vec<u8>.
+///
+/// # Optional Arguments
+///
+/// * `collection_ttl` - The time-to-live for the collection. If not provided, the client's default time-to-live is used.
+///
+/// # Examples
+/// Assumes that a CacheClient named `cache_client` has been created and is available.
+/// ```
+/// # fn main() -> anyhow::Result<()> {
+/// # use momento_test_util::create_doctest_client;
+/// # tokio_test::block_on(async {
+/// use momento::CollectionTtl;
+/// use momento::requests::cache::sorted_set_put_elements::SortedSetPutElements;
+/// use momento::requests::cache::sorted_set_put_elements::SortedSetPutElementsRequest;
+/// # let (cache_client, cache_name) = create_doctest_client();
+/// let sorted_set_name = "sorted_set";
+///
+/// let put_elements_request = SortedSetPutElementsRequest::new(
+///     cache_name.to_string(),
+///     sorted_set_name.to_string(),
+///     vec![("value1", 1.0), ("value2", 2.0)]
+/// ).with_ttl(CollectionTtl::default());
+///
+/// let put_elements_response = cache_client.send_request(put_elements_request).await?;
+///
+/// assert_eq!(put_elements_response, SortedSetPutElements {});
+/// # Ok(())
+/// # })
+/// # }
+pub struct SortedSetPutElementsRequest<S: IntoBytes, E: IntoBytes> {
+    cache_name: String,
+    sorted_set_name: S,
+    elements: Vec<(E, f64)>,
```

Comment:
I can imagine that we might want to support a Map type as input here, so we might need another Into thing, but we can explore that in a follow-up PR.